tests/nmstate_suite_test.go

```diff
@@ -18,26 +18,36 @@ package nmstate_tests
 
 import (
 	"flag"
+	"io/ioutil"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	yaml "github.com/ghodss/yaml"
+
+	apiappsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
-	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+	clientappsv1 "k8s.io/client-go/kubernetes/typed/apps/v1"
+	clientcorev1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/tools/clientcmd"
 
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
 
 	nmstate "github.com/nmstate/kubernetes-nmstate/pkg/client/clientset/versioned"
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/client/clientset/versioned/typed/nmstate.io/v1"
 )
 
 var kubeconfig *string
 var nmstateNs *string
 var manifests *string
+var nodeName string
 var k8sClientset *kubernetes.Clientset
 var nmstateClientset *nmstate.Clientset
-var nmstatePodsClient corev1.PodInterface
+var nmstatePodsClient clientcorev1.PodInterface
+var nmstateDaemonSetClient clientappsv1.DaemonSetInterface
+var defaultNNSs, nmstateNNSs nmstatev1.NodeNetworkStateInterface
```

Comment:
this section grows fast... we should think how to make it better, I don't want us to end up with a huge global junkyard